src/admin-views/tribe-options-display.php

```diff
@@ -3,28 +3,137 @@
  * @var array $settings List of display settings.
  */
 
-// Determine if ET was installed at version 5.0+.
-$should_default_to_on = ! tribe_installed_before( 'Tribe__Tickets__Main', '5.0' );
+/** @var Tribe__Dependency $dep */
+$dep = tribe( Tribe__Dependency::class );
 
-// Do not show the option for new installs.
-if ( $should_default_to_on ) {
+$etp_active = $dep->is_plugin_active( 'Tribe__Tickets_Plus__Main' );
+
+// Don't display this section if ET is a newer install and ETP is inactive or ETP is active but a newer install.
+if (
+	(
+		! $etp_active
+		&& ! tribe_installed_before( 'Tribe__Tickets__Main', '5.0' )
+	)
+	|| (
+		$etp_active
+		&& ! tribe_installed_before( 'Tribe__Tickets_Plus__Main', '5.1' )
+	)
+) {
 	return;
 }
 
-$settings = Tribe__Main::array_insert_before_key( 'tribe-form-content-end', $settings, [
-	'rsvp-display-title'         => [
+$tickets_rsvp_display_title = sprintf(
```

Comment:
late escape:

```php
esc_html(
  sprintf(
    _x(
      '%1$s Display Settings',
      '...',
      '...',
    ),
    tribe_get_rsvp_label_plural( '...' ),
  )
)
```

Reply:
@paulmskim - ah, i knew i was close. great catch. all should be fixed now.